src/js/components/Text/doc.js

```diff
@@ -58,6 +58,19 @@ is too long to all fit.`,
       PropTypes.oneOf(['normal', 'bold']),
       PropTypes.number,
     ]).description('Font weight'),
+    wordBreak: PropTypes.oneOf([
```

Comment:
Something we like to do for grommet is not follow directly the css properties with the aim to provide a better api.
So values such as inherit, initial, unset are probably not needed.
I also wonder whether it is better to have breakAll keepAll etc. This is more aligned with the properties in react ecosystem.

Reply:
I initially added global css values because I felt like they can also be used in certain cases. If you insist on removing them I will remove them.

I think having a camel case will help keep the code more clean as we already camel case other css properties.